lib/Runtime/Library/ConcatString.cpp

```diff
@@ -143,7 +143,10 @@ namespace Js
         this->propertyString = propStr;
         if (propStr != nullptr)
         {
-            this->propertyRecord = propStr->GetPropertyRecord();
+            // why use local variable ? read SWB wiki
```

Comment:
nit: This comment isn't really helpful. No comment is ok --  any reader interested or doubtful could easily find out there is a type mismatch between "this->propertyRecord" vs. "->GetPropertyRecord(...).